projects/media-viewer/src/lib/bookmark/bookmark.module.ts

```diff
@@ -0,0 +1,26 @@
+import { NgModule } from '@angular/core';
```

Comment:
not sure we need a separate module for bookmarks, especially not just for this component